### PR TITLE
fix(route): bilibili/live 修改areaLink默认值为empty（当areaLink为空时报错）

### PR DESCRIPTION
**Pull request by Shironobako**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by TonyRL** [comment]
Is there any routes that can be tested against this change?

**Comment by Shironobako** [comment]
https://rsshub.app/bilibili/live/area/339/online
https://rsshub.app/bilibili/live/area/555/online
https://rsshub.app/bilibili/live/area/371/online
https://rsshub.app/bilibili/live/area/367/online
https://rsshub.app/bilibili/live/area/373/online
https://rsshub.app/bilibili/live/area/561/online

File: lib/v2/bilibili/liveArea.js [hunk 1]
```diff
@@ -38,13 +38,15 @@ module.exports = async (ctx) => {
...
-                    case 2:
                     case 3:
                         areaLink = `https://live.bilibili.com/p/eden/area-tags#/${parentID}/${areaID}`;
                         break;
                     case 4:
                         areaLink = 'https://live.bilibili.com/pages/area/draw';
                         break;
+                    default:
+                        areaLink = 'empty'
+                        break;
```
Comment (TonyRL) [inline lib/v2/bilibili/liveArea.js:49]:
```suggestion
                    case 5:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    case 6:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    case 9:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    case 10:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    case 11:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    case 13:
                        areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
                        break;
                    default:
                        areaLink = 'empty';
                        break;
```

File: lib/v2/bilibili/liveArea.js [hunk 2]
```diff
@@ -34,18 +34,8 @@ module.exports = async (ctx) => {
...
-                    case 2:
-                    case 3:
-                        areaLink = `https://live.bilibili.com/p/eden/area-tags#/${parentID}/${areaID}`;
-                        break;
-                    case 4:
-                        areaLink = 'https://live.bilibili.com/pages/area/draw';
-                        break;
-                }
+                areaLink = `https://live.bilibili.com/p/eden/area-tags?parentAreaId=${parentID}&areaId=${areaID}`;
+                break;
```
[the remark this answers is not in the document]
Comment (Shironobako) [reply lib/v2/bilibili/liveArea.js:38]: 这个break是退出上一层for-loop

**Pull request merged**